opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectOverview.jsx

```diff
@@ -154,6 +155,39 @@ class StixDomainObjectOverview extends Component {
               <ItemAuthor
                 createdBy={R.propOr(null, 'createdBy', stixDomainObject)}
               />
+              <Grid container={true} columnSpacing={1}>
+                <Grid item={true} xs={6}>
+                  <Typography
+                    variant="h3"
+                    gutterBottom={true}
+                    style={{ marginTop: 20 }}>
+                    {t('Reliability')}
+                    {!stixDomainObject.x_opencti_reliability && (
+                      <span style={{ fontStyle: 'italic' }}> ({t('of author')})</span>
+                    )}
+                  </Typography>
+                  <ItemOpenVocab
+                    key="type"
+                    type="reliability_ov"
+                    value={!stixDomainObject.x_opencti_reliability ? stixDomainObject.createdBy?.x_opencti_reliability : stixDomainObject.x_opencti_reliability}
```

Comment:
Reliability on individual seems to not working

My author
![image](https://github.com/OpenCTI-Platform/opencti/assets/53513584/403d2f98-e704-4fc8-ad4a-11ed10e1f9a3)

An other user
![image](https://github.com/OpenCTI-Platform/opencti/assets/53513584/0740b511-7672-4dfa-b88f-99c76ec20c97)
![image](https://github.com/OpenCTI-Platform/opencti/assets/53513584/ab7cd480-1889-4f56-ab24-f39600c0f310)

Reply:
It's fixed now